src/tool_setup.sh

```diff
@@ -4,15 +4,14 @@ sudo apt-get update
 sudo apt-get install -y make
 sudo apt-get install -y python3-venv
```

Comment:
Co odpalamy w venv'ie?

Reply:
Wcześniej odpalalismy serwer HTTP. Teraz jest to na dobra sprawę jedynie furtka do dalszych tooli do ataków/obrony